contracts/BodhiToken.sol

```diff
@@ -70,7 +70,8 @@ contract BodhiToken is StandardToken, Ownable {
   function buyTokens(address _beneficiary) 
     payable 
     validAddress(_beneficiary) 
-    validPurchase {
+    validPurchase
+  {
```

Comment:
![image](https://user-images.githubusercontent.com/29769992/30001232-78fe3106-903c-11e7-8da7-4ff5f75a49ca.png)

The left paren should be on its own line when there is modifiers.